### ISSUE TITLE
Export fails if podcast name has a forward slash

### ISSUE THREAD

**Issue by jzohrab** [hagent/export-macos-podcasts#4]
Currently the export fails when I try to export, as one of my podcasts is named "La Segunda Guerra Mundial (E/P/T)".  The forward slash in the title causes mkdir to fail:

```
node:fs:1357
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, mkdir '/Users/jeff/Downloads/PodcastsExport/2022.05.12/La Segunda Guerra Mundial (E/P/T)'
... snip  syscall: 'mkdir',
  code: 'ENOENT',
  path: '/Users/jeff/Downloads/PodcastsExport/2022.05.12/La Segunda Guerra Mundial (E/P/T)'
}
```

**Comment by jzohrab** [comment]
This is an easy fix.  I'll just substitute `/` with `_`, resulting in the name `La Segunda Guerra Mundial (E_P_T)`.  Sending in a PR now.

Thank you for this project!

**Comment by jzohrab** [comment]
Also fixed in #6, which I'll re-push soon.

**Comment by jzohrab** [comment]
Fixed in #6.
